casts/rows/src/UILink/UILink.tsx

```diff
@@ -59,6 +59,7 @@ const styles = StyleSheet.create({
     },
     textContent: {
         flex: 1,
+        borderRadius: 0,
```

Comment:
What this for? AFAIR it's 0 by default. What is it doing?
Need to at least leave a comment, to not delete it along the way, as it seems unnecessary without the context.

Reply:
Removed, no longer required